app/controllers/api/v1/npq_participants_controller.rb

```diff
@@ -29,6 +29,12 @@ def withdraw
         end
       end
 
+      def defer
+        service = DeferParticipant.new(params_for_recorder)
```

Comment:
Since we have to live in both worlds of `participant_actions` vs new actions, we will need to reuse some of those components such as `params_for_recorder` etc. however when we move all actions to the new world, we can get rid of participant actions and have have explicit methods for both NPQ and ECF